go-runtime/compile/schema_test.go

```diff
@@ -367,10 +434,13 @@ func TestErrorReporting(t *testing.T) {
 		`76:63-63: second result must not be ftl.Unit`,
 		`83:1-1: duplicate verb name "WrongResponse"`,
 		`89:2-12: struct field unexported must be exported by starting with an uppercase letter`,
-		`100:2-23: cannot attach enum value to BadValueEnum because it is a variant of type enum TypeEnum, not a value enum`,
-		`106:2-40: cannot attach enum value to BadValueEnumOrderDoesntMatter because it is a variant of type enum TypeEnum, not a value enum`,
-		`115:1-26: parent enum "ExportedTypeEnum" is exported, but directive "ftl:data" on "PrivateData" is not: all variants of exported enums that have a directive must be explicitly exported as well`,
-		`119:21-60: config and secret names must be valid identifiers`,
+		`101:2-24: cannot attach enum value to BadValueEnum because it is a variant of type enum TypeEnum, not a value enum`,
+		`108:2-41: cannot attach enum value to BadValueEnumOrderDoesntMatter because it is a variant of type enum TypeEnum, not a value enum`,
+		`117:1-26: parent enum "ExportedTypeEnum" is exported, but directive "ftl:data" on "PrivateData" is not: all variants of exported enums that have a directive must be explicitly exported as well`,
+		`121:21-60: config and secret names must be valid identifiers`,
+		`127:1-26: only one directive expected for type alias`,
+		`133:2-2: type is not declared as an ftl type alias or enum: NonFTLAlias`,
```

Comment:
I was a bit confused by this error message at first. I wonder if shuffling it around would make it more clear 🤔 

```
type "NonFTLAlias" is not declared as an ftl type alias or enum 
```

Reply:
Agreed, but also in general I tend to leave out the typename iff the error is pointing directly at it already, as it's redundant information (not sure if that is the case here).

Reply:
Switched it to:
> type is not declared as an ftl enum or type alias